idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -1103,13 +1236,16 @@ def bubble_cloud_bulk_heat_trans(b, t, x):
                 conc_diff = (
                         b.gas_emulsion.properties[t, x].dens_mol -
                         b.bubble.properties[t, x].dens_mol)
+                conc_diff_units = (units_meta_gas('amount'))
                 return (
                     b.Hgbulk[t, x] * b.bubble_diameter[t, x] ==
-                    6 * b.Kd * b.delta[t, x] * b.bed_area *
+                    6 * b.Kd * b.delta[t, x] *
+                    pyunits.convert(b.bed_area,
+                                    to_units=units_meta_gas('area')) *
                     (b.gas_emulsion.properties[t, x].enth_mol *
-                     smooth_max(conc_diff, 0, b.eps) +
+                     smooth_max(value(conc_diff), 0, b.eps)*conc_diff_units +
```

Comment:
This will not work - `value` returns a single value using the current value of the expression, so this effectively becomes a constant rather than an expression.

In this case, both `0` and `eps` should have the same units as `conc_diff` and you should just do `smooth_max(conc_diff, 0, b.eps)`

Reply:
I updated how the `eps` Param objects are defined to ensure unit consistency when we call the smoothing factors.